### PR TITLE
Implement header component

### PR DESCRIPTION
**Pull request by zuziasobanska**
Header component with responsiveness, colour and font variables added

### CODE REVIEW

File: PolishWithBlondes/src/App.tsx [hunk 1]
```diff
@@ -1,10 +1,18 @@
+import { Home } from './pages/home/Home';
+import { Navbar } from './components/navbar/Navbar';
+
+import './styles/reset.css'; // Import your reset CSS here
+import './styles/global.css'; // Import global CSS
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/App.tsx]: Comments not needed

File: PolishWithBlondes/src/App.tsx [hunk 2]
```diff
@@ -1,10 +1,18 @@
...
 import './App.scss';
 
 function App() {
   return (
-    <div className="container">
-      <h1>Polish with Blondes</h1>
-    </div>
+    <>
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/App.tsx]: Do we still need React Fragment here?

File: PolishWithBlondes/src/styles/reset.css [hunk 3]
```diff
@@ -0,0 +1,40 @@
+/* reset.css */
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/styles/reset.css]: Not needed

File: PolishWithBlondes/src/styles/reset.css
Comment (pawel-peksa) [inline PolishWithBlondes/src/styles/reset.css:1]: Let's stick to scss files

File: PolishWithBlondes/src/styles/global.css [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+$secondary-font-color: #ff7755; // Warm reddish-orange
+
+Secondary background
+  --secondary-background: #fff; // Clean white
+
+Accent color (maybe the same as secondary font/background)
+  --accent-color: #ff7755; // Consistent accent color
+
+
+ */
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/styles/global.css]: Do we need this?

File: PolishWithBlondes/src/styles/global.css
Comment (pawel-peksa) [inline PolishWithBlondes/src/styles/global.css:1]: Change to scss

File: PolishWithBlondes/src/App.scss [hunk 5]
```diff
@@ -1,12 +1,18 @@
-$primary-color: #3498db;
+* {
+  border: 1px solid red;
+}
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/App.scss]: It's good that you use it for development purposes but please don't merge it to the main branch. (Remove this before each PR)

File: PolishWithBlondes/src/App.scss [hunk 6]
```diff
@@ -1,12 +1,18 @@
...
-  color: white;
-  padding: 20px;
-  text-align: center;
+.app-container {
+  background-color: var(--primary-background);
 }
 
-h1 {
-  font-size: 2rem;
+@media (max-width: 1000px) {
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/App.scss]: Not sure if that is possible, but let's try define constant break points and refer to them in media queries

File: PolishWithBlondes/src/components/navbar/Navbar.tsx [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+import hamburger from '../../assets/hamburger.png';
+
+export const Navbar = () => {
+  return (
+    <nav className="navbar">
+      <img src={logo} className="logo" />
+      <ul>
+        <li>
+          <a href="">Book a lesson</a>
+        </li>{' '}
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/Navbar.tsx]: Please delete these `{' '}`

File: PolishWithBlondes/src/components/navbar/Navbar.tsx [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+        <li>
+          <a href="">Worksheets</a>
+        </li>{' '}
+        <li>
+          <a href="">Podcasts</a>
+        </li>{' '}
+        <li>
+          <a href="https://www.youtube.com/@polishwithblondes">Youtube</a>
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/Navbar.tsx]: Let's make it open in a new tab

File: PolishWithBlondes/src/components/navbar/Navbar.tsx [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+        </li>
+        <li>
+          <a href="https://buymeacoffee.com/polishwithblnds">
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/Navbar.tsx]: Same here, new tab

File: PolishWithBlondes/src/components/navbar/Navbar.tsx [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+        </li>
+        <li>
+          <a href="">About us</a>
+        </li>
+        <li>
+          <a href="">Contact</a>
+        </li>
+      </ul>
+      <div className="hamburger">
+        <img src={hamburger} />
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/Navbar.tsx:37]: Later on, maybe we could use a Hamburger component from the npm

File: PolishWithBlondes/src/components/navbar/navbar.scss [hunk 11]
```diff
@@ -0,0 +1,42 @@
...
+  display: flex;
+  justify-content: space-between;
+  max-width: 1060px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.navbar ul {
+  display: flex;
+  flex-direction: row;
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/navbar.scss]: that is a default value, no need to set it

File: PolishWithBlondes/src/components/navbar/navbar.scss [hunk 12]
```diff
@@ -0,0 +1,42 @@
...
+  list-style: none;
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/navbar.scss]: not needed - you have it in reset.css

File: PolishWithBlondes/src/components/navbar/navbar.scss [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+.navbar li {
+  align-self: center;
+  margin: 0px 5px;
+}
+
+.navbar a {
+  color: var(--primary-font-color);
+  text-decoration: none;
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/navbar.scss:28]: Let's put this to reset.scss

File: PolishWithBlondes/src/components/navbar/navbar.scss [hunk 14]
```diff
@@ -0,0 +1,42 @@
...
+  text-transform: uppercase;
+}
+
+.logo {
+  padding: 0;
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/components/navbar/navbar.scss]: is it needed? Padding should be 0 thanks to reset.scss if not. set it to 0 for everything there

File: PolishWithBlondes/src/index.css
Comment (pawel-peksa) [inline PolishWithBlondes/src/index.css:1]: Delete it if not needed

File: PolishWithBlondes/src/pages/home/Home.tsx [hunk 15]
```diff
@@ -0,0 +1,9 @@
+export const Home = () => {
+  return (
+    <>
```
Comment (pawel-peksa) [inline PolishWithBlondes/src/pages/home/Home.tsx]: Do we need it?

**Review by pawel-peksa — CHANGES_REQUESTED** [review]
Good work. I've added a couple of minor comments.

File: PolishWithBlondes/src/App.tsx (hunk 1 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/App.tsx]: Resolved

File: PolishWithBlondes/src/App.tsx (hunk 2 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/App.tsx]: No we don't. Resolved

File: PolishWithBlondes/src/styles/reset.css (hunk 3 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/styles/reset.css]: Ok, resolved

File: PolishWithBlondes/src/styles/reset.css
Comment (zuziasobanska) [reply PolishWithBlondes/src/styles/reset.css:1]: Ok. Resolved

File: PolishWithBlondes/src/styles/global.css (hunk 4 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/styles/global.css]: No. Resolved

File: PolishWithBlondes/src/styles/global.css
Comment (zuziasobanska) [reply PolishWithBlondes/src/styles/global.css:1]: Resolved

File: PolishWithBlondes/src/App.scss (hunk 5 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/App.scss]: Will do. Resolved

File: PolishWithBlondes/src/App.scss (hunk 6 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/App.scss]: Good idea, I'll remember this for the future

File: PolishWithBlondes/src/components/navbar/Navbar.tsx (hunk 7 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/Navbar.tsx]: Resolved

File: PolishWithBlondes/src/components/navbar/Navbar.tsx (hunk 8 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/Navbar.tsx]: Resolved

File: PolishWithBlondes/src/components/navbar/Navbar.tsx (hunk 9 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/Navbar.tsx]: resolved

File: PolishWithBlondes/src/components/navbar/Navbar.tsx (hunk 10 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/Navbar.tsx:37]: Ok, let's do that later on

File: PolishWithBlondes/src/components/navbar/navbar.scss (hunk 11 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/navbar.scss]: resolved

File: PolishWithBlondes/src/components/navbar/navbar.scss (hunk 12 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/navbar.scss]: resolved

File: PolishWithBlondes/src/components/navbar/navbar.scss (hunk 13 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/navbar.scss:28]: done

File: PolishWithBlondes/src/components/navbar/navbar.scss (hunk 14 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/components/navbar/navbar.scss]: Yes, that's true. Resolved

File: PolishWithBlondes/src/index.css
Comment (zuziasobanska) [reply PolishWithBlondes/src/index.css:1]: resolved

File: PolishWithBlondes/src/pages/home/Home.tsx (hunk 15 above)
Comment (zuziasobanska) [reply PolishWithBlondes/src/pages/home/Home.tsx]: no. resolved

**Review by pawel-peksa — CHANGES_REQUESTED** [review]
Good job. One more thing, please add
.vscode/*
line to .gitignore and remove from your commit all the files that are in this PR

**Review by pawel-peksa — APPROVED** [review]
Nice! 
ChiquichicoGIF

**Pull request merged**